doc/source/catalog_generation.rst

```diff
@@ -359,6 +359,84 @@ ordering in the .ecsv file as well):
+effort to only publish catalogs which provide the highest science value, criteria developed by the Hubble Legacy
+Archive (HLA) has been implemented to recognize those catalogs dominated by cosmic-rays and not provided as an
+output product.
+
+.. note ::
+  This rejection criteria is NOT applied to WFC3/IR or ACS/SBC data since they are not affected by cosmic-rays
+  in the same way as the other detectors.
+
+3.3.1 Rejection Criteria
+"""""""""""""""""""""""""
+The rejection criteria has been defined so that if either the point source catalog or the segment catalog fails,
+then both catalogs are rejected and deleted.
+
+In its simplest form the criteria for rejection is:
+        n_cat < thresh
+where:
+        thresh = crfactor * (n1_residual * n1_exposure_time)**2 / texptime
+and:
+        n_cat    : Number of good point and extended sources in the catalog (flag < 2)
+        crfactor : Number of expected cosmic-rays per second across the entire detector
+        n1_exposure_time : amount of exposure time for all single filter exposures
+        texptime : Total exposure time of the combined drizzle product
+        n1_residual : Remaining fraction of cosmic-rays after applying single-image CR removal
+
+The value of `crfactor` should be adjusted for sub-arrays to account for the smaller area being read out, but
+that logic has not yet been implemented.  The values used in the processing of single-visit mosaics are:
+
+    segment-catalog crfactor : 300
+    point-catalog crfactor   : 150
+
+These numbers are deliberately set high to be conservative about which catalogs to keep.  The CR rate varies
+with position in the orbit, and these are set high enough that it is rare for approved catalogs to be dominated
+by CRs (even though they can obviously have some CRs included.)
+
+Finally, the `n1_residual` term gets set as a configuration parameter with a default value of 5% (0.05).  This
+indicates that the single-image cosmic-ray identification process was expected to leave 5% of the cosmic-rays
+unflagged. This process can be affected by numerous factors, and having this as a user settable parameter allows
+the user to account for these effects when reprocessing the data manually.  Pipeline processing, though, may
+still be subject to situations where this process does not do as well which can result in a catalog with a
+higher than expected contribution of cosmic-rays.  Should this number of sources trigger the rejection criteria,
+these catalogs will be rejected and not written out.
+
+Also note that we reject both the point and segment catalogs if either one fails this test.  The reasoning
+behind that is that since the catalogs are based on the same image, it is unlikely that one catalog will be
+good and the other contaminated.
+
+Should the catalogs fail this test, neither type of catalogs will be written out to disk for this visit.
+
+
+3.3.2 Single-image CR Rejection
+""""""""""""""""""""""""""""""""
```

Comment:
I would add the word Algorithm to the title of this subsection.

Reply:
Done in 4bd2200.